doc/source/user_guide/timeseries.rst

```diff
@@ -1264,11 +1264,16 @@ frequencies. We will refer to these aliases as *offset aliases*.
     "BAS, BYS", "business year start frequency"
     "BH", "business hour frequency"
     "H", "hourly frequency"
-    "T, min", "minutely frequency"
+    "min", "minutely frequency"
     "S", "secondly frequency"
-    "L, ms", "milliseconds"
-    "U, us", "microseconds"
-    "N", "nanoseconds"
+    "ms", "milliseconds"
+    "us", "microseconds"
+    "ns", "nanoseconds"
+
+.. deprecated:: 2.1.0
+
+   Aliases ``T``, ``L``, ``U``, and ``N`` are deprecated in favour of the aliases
+   ``min``, ``ms``, ``us``, and ``ns``.
```

Comment:
This looks fine to me, I think it's a better experience if the list of aliases above teaches the new behaviour to begin with - but OK with adding `T` / `L` etc. back there if people prefer